synthtool/gcp/templates/node_library/samples/README.md

```diff
@@ -0,0 +1,51 @@
+[//]: # "This README.md file is auto-generated, all changes to this file will be lost."
+[//]: # "To regenerate it, use `npm run generate-scaffolding`."
+<img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google Cloud Platform logo" title="Google Cloud Platform" align="right" height="96" width="96"/>
+
+# [{{ metadata['repo']['name_pretty'] }}: {{ metadata['repo']['language']|language_pretty }} Client](https://github.com/{{ metadata['repo']['repo'] }})
+
+[![Open in Cloud Shell][shell_img]][shell_link]
+
+{{ metadata['partials'] and metadata['partials']['introduction'] }}
+
+## Table of Contents
+
+* [Before you begin](#before-you-begin)
+* [Samples](#samples){% if metadata['samples']|length %}{% for sample in metadata['samples'] %}
+  * [{{ sample.name }}](#{{ sample.name|slugify }}){% endfor %}{% endif %}
+
+## Before you begin
+
```

Comment:
this README has been simplified, the need for executing samples to generate `yargs` output, this matches the direction we're moving in for samples any ways.